src/templates/import.js

```diff
@@ -152,20 +152,22 @@ module.exports = (router) => {
       return formRevision.toString();
     };
 
-    const formName = entity.form;
+    let formName = entity.form;
+    if (template.forms && !template.forms[formName] && entity.form.length === 24) {
+      formName = _.findKey(template.forms, (form) => {
+        return form._id.toString() === entity.form;
```

Comment:
This makes me believe that the export has an issue. No id's should make it into the template.json file. Can we fix the problem at the source instead of having to write the import process to be able to handle ID's that should not be there?